packages/multi_test/tests/multi_test/rujira_invalid_price/helper.rs

```diff
@@ -0,0 +1,149 @@
+use cosmwasm_std::{
+    testing::MockQuerier, Addr, Binary, GrpcQuery, Querier, QueryRequest, SystemResult,
+};
+use cw_multi_test::{App, AppBuilder};
+use cw_storage_plus::Item;
+use mocktail::{server::MockServer, MockSet};
+use perpswap::namespace;
+use prost::Message;
+use rujira_rs::proto::types::{QueryPoolRequest, QueryPoolResponse};
+
+pub const FACTORY_ADDR: Item<Addr> = Item::new(namespace::FACTORY_ADDR);
+
+pub const _GOVERNANCE: &str = "cosmwasm1h72z9g4qf2kjrq866zgn78xl32wn0q8aqayp05jkjpgdp2qft5aquanhrh";
+pub const MARKET_ADDR1: &str =
+    "cosmwasm1qnufjmd8vwm6j6d3q28wxqr4d8408f34fpka4vs365fvskualrasv5ues5";
+pub const _MARKET_ADDR2: &str =
+    "cosmwasm1vqjarrly327529599rcc4qhzvhwe34pp5uyy4gylvxe5zupeqx3sg08lap";
+pub struct CustomGrpcQuerier {
```

Comment:
Can we have just one implementation of this instead of multiple copies in the code?